src/main/java/seedu/duke/utils/Ui.java

```diff
@@ -180,15 +192,15 @@ public static void printLine() {
     }
 
     public static void printDoubleNeeded() {
-        System.out.println(LINE);
+        printLine();
```

Comment:
small issue but does this count as not following SLAP? from what i understand the purpose of `printLine()` is to be used in other classes so we don't have situations like
```
Ui.printSomething();
System.out.println(LINE); //different level of abstraction than the line above
```

for the methods within the Ui class, i'm not sure if it's necessary to replace everything with `printLine()`.

Reply:
Not too sure about this, because when I pulled from the repo there were some functions using printLine() but others were using System.out.println(LINE), let me know which one works better, I can just change them out.

Reply:
i think it's safer to do `System.out.println(LINE)` but not sure if this really matters anyway